src/shogun/evaluation/SigmoidCalibration.cpp

```diff
@@ -42,28 +42,75 @@
 
 using namespace shogun;
 
-CSigmoidCalibration::CSigmoidCalibration() : CCalibration()
+CSigmoidCalibration::CSigmoidCalibration()
 {
 	init();
 }
 
 CSigmoidCalibration::~CSigmoidCalibration()
 {
-	SG_FREE(m_sigmoid_parameters);
 }
 
 void CSigmoidCalibration::init()
 {
-	m_sigmoid_parameters = SG_MALLOC(CStatistics::SigmoidParamters, 1);
+	m_sigmoid_parameters.resize(1);
+	m_maxiter = 100;
+	m_minstep = 1E-10;
+	m_sigma = 1E-12;
+	m_epsilon = 1E-5;
+
```

Comment:
`m_sigmoid_parameters` is not registered... that'll make it impossible to clone this class. since currently registering std::vector is not possible you would need to use DynArray :(

Reply:
Thanks for telling me about `DynArray`, I was looking for the standard shogun way of storing arrays of non-CSGObjects.

Reply:
I have opted to use two SGVectors for now

Reply:
Think that should be OK as a workaround for now, especially since the number of parameter is fixed

Reply:
actually, even better would be to put the parameters explicitly. The vector length should not be variable